src/main/java/nextstep/subway/domain/path/PathFinderStrategy.java

```diff
@@ -0,0 +1,10 @@
+package nextstep.subway.domain.path;
+
+import nextstep.subway.domain.Line;
+import nextstep.subway.domain.Station;
+
+import java.util.List;
+
+public interface PathFinderStrategy {
+    public Path findShortestPathAndItsDistance(List<Line> lines, Station sourceStation, Station targetStation);
```

Comment:
```suggestion
    Path findShortestPathAndItsDistance(List<Line> lines, Station sourceStation, Station targetStation);
```

interface 는 public 을 명시하지 않아도 public method 로 선언됩니다